test/README.md

```diff
@@ -67,3 +95,14 @@ make shell
 ```
 
 This will run a container and give you a shell and you can follow the instructions above.
+
+# System test
+System test is used for testing podman command line with entire system. It requests podman
+command line installed and configured like it is used in an user enviroment.
```

Comment:
Suggestion:

```markdown
System tests are used for testing the *podman* CLI in the context of a complete system. It requests that
*podman*, all dependencies, and configurations are in place.  The intention of system testing, is to match
as closely as possible, with real-world user/developer use-cases and environments.  Though orchestration
of the environments and tests is left to external tooling.
```

Renders as:

System tests are used for testing the *podman* CLI in the context of a complete system. It requests that
*podman*, all dependencies, and configurations are in place.  The intention of system testing, is to match
as closely as possible, with real-world user/developer use-cases and environments.  Though orchestration
of the environments and tests is left to external tooling.